protocol/contracts/templegold/AuctionBase.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity 0.8.20;
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Temple (templegold/AuctionBase.sol)
+
+import { IAuctionBase } from "contracts/interfaces/templegold/IAuctionBase.sol";
+
+abstract contract AuctionBase is IAuctionBase {
+    /// @notice Current epoch id
+    uint256 internal _currentEpochId;
+    /// @notice Keep track of epochs details
+    mapping(uint256 epochId => EpochInfo info) internal epochs;
+    /// @notice Keep track of depositors for each epoch
+    mapping(address depositor => mapping(uint256 epochId => uint256 amount)) public override depositors;
```

Comment:
You can remove `override` as it does not override anything

Reply:
We use `override` here for all public variables and functions defined in the interface (`IAuctionBase`). This makes the compiler throw an error if we actually haven't implemented a function from the interface.